ethereum/package.json

```diff
@@ -51,11 +51,18 @@
     "build": "hardhat compile",
     "clean": "hardhat clean",
     "test": "CONTRACT_TESTS=1 yarn hardhat test test/unit_tests/*.spec.ts --network hardhat",
+    "test:foundry": "hardhat solpp && forge test",
     "test:fork": "CONTRACT_TESTS=1 TEST_CONTRACTS_FORK=1 yarn run hardhat test test/unit_tests/*.fork.ts --network hardhat",
     "lint": "yarn lint:sol && yarn prettier:check",
-    "lint:sol": "solhint --max-warnings 29 contracts/**/*.sol",
-    "prettier:check": "prettier --check contracts/**/*.sol",
-    "prettier:write": "prettier --write contracts/**/*.sol",
+    "lint:sol-contracts": "solhint --max-warnings 29 contracts/**/*.sol",
```

Comment:
hould we remove hardcoded "29" for max allowed warnings? Or the plan is to update it whenever we update contracts and know why the warning is emitted?